2022q1/office.adoc

```diff
@@ -0,0 +1,25 @@
+=== FreeBSD Office Team
+
+Links: +
+link:https://wiki.freebsd.org/Office[The FreeBSD Office project] URL: link:https://wiki.freebsd.org/Office[https://wiki.freebsd.org/Office]
+link:https://lists.freebsd.org/subscription/freebsd-office[The FreeBSD Office mailing list] URL: link:https://lists.freebsd.org/subscription/freebsd-office[https://lists.freebsd.org/subscription/freebsd-office]
+
+Contact: FreeBSD Office team ML <office@FreeBSD.org> +
+Contact: Dima Panov <fluffy@FreeBSD.org> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org>
+
+The FreeBSD Office team works on a number of office-related software suites and tools such as OpenOffice and LibreOffice.
+
+Work during this quarter was focused on providing the latest stable release of LibreOffice suite and companion apps to all FreeBSD users.
+
+During the 2022Q1 period we pushed maintenance patches for LibreOffice 7.2 leaf to the quarterly branch and bring the latest, 7.3, releases
```

Comment:
s/bring/brought/

Also, "leaf"? Not sure what you mean by that.

Reply:
@fluffykhv: maybe just call it "Libreoffice 7.2 port" ?

Reply:
Ah, yes. Sorry for misunderstood guys, last week was hard IRL, my mind is blowing :(